generative_model_training_all_labels/05_plot_labels_sampling_performance_all_labels.ipynb

```diff
@@ -0,0 +1,1635 @@
+{
```

Comment:
<p>I think you mention it somewhere else, but you may want to explain why you selected these as the cutoffs for the number of label functions used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/snorkeling-full-text/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>